components/common/layouts/dashboard.js

```diff
@@ -160,11 +164,13 @@ const withDashboardLayout = (children) => {
 						}
 
 						const setStateAndTrack = (details) => {
-							trackEvent(Events.USER_ACCOUNT_SELECTION, {
-								user: {
-									...details,
-									stashId: address,
-								},
+							setUserProperties({
+								selectedStashId: address,
```

Comment:
@saumyakaran 
different key names used, in _base.js_ `stashId` and here `selectedStashId`

Reply:
Updated https://github.com/buidl-labs/yieldscan-frontend/pull/127/commits/0a21b6e3ac0fb4fac08a0d6b4d0d05724b92de63